gazebo_msgs/srv/StartTimer.srv

```diff
@@ -0,0 +1,5 @@
+float64 duration_write_joint_states
```

Comment:
comment documentation? also, this service seems a little too application-specific